### PR TITLE
api as sub mod

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

File: core/cafe_api_v1.go [hunk 1]
```diff
@@ -631,3 +631,18 @@ func (c *cafeApi) search(g *gin.Context) {
...
+
+// pbJSON responds with a JSON rendered protobuf message
+func pbJSON(g *gin.Context, status int, msg proto.Message) {
+	str, err := pbMarshaler.MarshalToString(msg)
+	if err != nil {
+		sendError(g, err, http.StatusBadRequest)
+		return
+	}
+	g.Data(status, "application/json", []byte(str))
+}
```
Comment (andrewxhill) [inline core/cafe_api_v1.go]: cafe_api will obv need to be it's own mod soon too, for now i just made this work as is

File: go.mod [hunk 2]
```diff
@@ -55,7 +58,8 @@ require (
 	github.com/xeipuuv/gojsonschema v1.1.0
 	go.uber.org/fx v1.9.0
 	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
-	google.golang.org/appengine v1.4.0 // indirect
+	google.golang.org/api v0.10.0 // indirect
+	google.golang.org/appengine v1.5.0
```
Comment (andrewxhill) [inline go.mod]: that wasn't intentional... removing

Comment (andrewxhill) [reply go.mod]: take that back, it was already there

Comment (sanderpick) [reply go.mod]: something keeps adding that appengine dep, keep forgetting to look into it, oh well

**Pull request merged**